tests/src/utils.rs

```diff
@@ -816,6 +819,18 @@ impl KubeService {
         format!("deployment/{}", self.name)
     }
 
+    pub fn rollout_target(&self) -> String {
+        if self.rollout.is_none() {
+            panic!("Rollout is not enabled for this service! Don't you mean to use a deployment?");
+        }
+
+        format!("rollout/{}", self.name)
+    }
+
+    pub fn rollout_t(&self) -> Option<&Rollout> {
```

Comment:
```suggestion
    pub fn rollout(&self) -> Option<&Rollout> {
```
Also, the field is public. Do you even need this function? :eyes:

Reply:
Nope, removed.